ColorPatterns/Scenes/Info/InfoController.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  InfoController.swift
+//  ColorPatterns
+//
+//  Created by Marek Skrzelowski on 23/02/2020.
+//  Copyright © 2020 vandermesis. All rights reserved.
+//
+
+import UIKit
+
+final class InfoController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/96a30a06f734813aa949b981c6177532a5bc762f/ColorPatterns/Scenes/Info/InfoController.swift#L15"></span>Override methods which only call super can be removed</td>
    </tr>
  </tbody>
</table>